clang/test/SemaCXX/constexpr-frame-describe.cpp

```diff
@@ -67,15 +68,19 @@ struct Bar {
   template <typename U, int num>
   constexpr int fail2() const { return 1 / 0; } // expected-warning {{division by zero}} \
                                                 // expected-note {{division by zero}}
+#ifndef NEW_CONST_INTERP
   template <typename ...Args>
   constexpr int fail3(Args... args) const { return 1 / 0; } // expected-warning {{division by zero}} \
                                                 // expected-note {{division by zero}}
```

Comment:
Seems we missed an `division by zero` diagnostic here.

Reply:
The output looks the same here: https://godbolt.org/z/PP868Eh87

Reply:
Ah, I get it. It's missing because the `static_assert` calling that function is `#if 0`ed out.

Reply:
Wow! Thanks! Sorry, this is my mistake, somehow I missed it.